bubble_sort/Java/BubbleSortRecursive.java

```diff
@@ -0,0 +1,46 @@
+public class BubbleSortRecursive {
+    public static void main(String[] args){
+    
+    //Taking a sample array for sorting it.
+        int[] nums = {12, 45, 23, 78, 67};
+
+    //calling the method and passing the array in it.
+        bubble(nums, nums.length);
+	
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpddmzpmr6/bubble_sort/Java/BubbleSortRecursive.java
+++ b/tmp/tmpddmzpmr6/bubble_sort/Java/BubbleSortRecursive.java
@@ -1,6 +1,6 @@
 public class BubbleSortRecursive {
     public static void main(String[] args){
-    
+
     //Taking a sample array for sorting it.
         int[] nums = {12, 45, 23, 78, 67};
 
```